app/routes/chatbot.js

```diff
@@ -253,6 +253,12 @@ router.use((req, res, next) => {
  * Track incoming message (and outgoing, if this is a reply to a broadcast).
  */
 router.use((req, res, next) => {
+  // If this is a retry, we don't want to track duplicate analytics for this request.
+  const retryCount = Number(req.headers['x-blink-retry-count']);
```

Comment:
It's better to use [`req.get()`](https://expressjs.com/en/api.html#req.get) to access headers, as it performs the case-insensitive search.